.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: java
 dist: trusty
 script:
-  - ./gradlew build && ./gradlew codeCoverageReport
+  - ./gradlew build && ./gradlew intTest && ./gradlew codeCoverageReport
```

Comment:
Doesn't codeCoverageReport force all tests to be run?  I would think your gradle changes make the intTest here unnecessary.